etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -126,23 +122,15 @@ public Map<HttpEndpoint, Function<DomainRequest, DomainResponse>> domainRegistra
                         DatabaseCredentialsProvider.class,
```

Comment:
Let's move the `DatabaseCredentialsProvider` and it's implementation registration into `App.java`.

Reply:
Fixed